kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -431,6 +443,7 @@ window.ExerciseView = Backbone.View.extend({
             this.related_video_view.remove();
         }
         this.$("input").qtip("destroy", /* immediate */ true);
+        this.closed = true;
```

Comment:
Yep, definitely worth doing when we refactor all of the Backbone View stuff to use our BaseView.